application/modules/settings/views/partial_settings_invoices.php

```diff
@@ -81,6 +81,38 @@ class="form-control" value="<?php echo get_setting('invoices_due_after'); ?>">
                     </div>
                 </div>
 
+                <div class="row">
+                    <div class="col-xs-12 col-md-6">                
+                        
+                        <div class="form-group">
+                            <label for="settings[default_creditnote_group]">
+                                <?php _trans('default_creditnote_group'); ?>
+                            </label>
+                            <select name="settings[default_creditnote_group]" id="settings[default_creditnote_group]"
+                                class="form-control simple-select" data-minimum-results-for-search="Infinity">
+                                <option value=""><?php _trans('none'); ?></option>
+                                <?php foreach ($invoice_groups as $invoice_group) { ?>
+                                    <option value="<?php echo $invoice_group->invoice_group_id; ?>"
+                                        <?php check_select(get_setting('default_creditnote_group'), $invoice_group->invoice_group_id); ?>>
+                                        <?php echo $invoice_group->invoice_group_name; ?>
+                                    </option>
+                                <?php } ?>
+                            </select>
+                        </div>
+
+                    </div>
+                    <div class="col-xs-12 col-md-6"> 
+                        <div class="form-group">
+                            <label for="settings[default_creditnote_terms]">
+                                <?php _trans('default_creditnote_terms'); ?>
+                            </label>
+                            <textarea name="settings[default_creditnote_terms]" id="settings[default_creditnote_terms]"
+                                class="form-control" rows="1"><?php echo get_setting('default_creditnote_terms', '', true); ?>
```

Comment:
Please use 4 rows as for the invoice notes.

Reply:
Changed <textarea> to 4 rows.